src/pages/library/Showcase.js

```diff
@@ -45,7 +45,10 @@ const makeCard = variant => ({ workspace: { namespace, name, attributes: { descr
         backgroundRepeat: 'no-repeat', backgroundPosition: 'center', backgroundSize: 'auto 100%', borderRadius: '0 5px 5px 0',
         width: 87,
         ...Utils.switchCase(variant,
-          ['new', () => ({ backgroundImage: `url(${covidBg})` })],
+          ['new', () => Utils.switchCase(
```

Comment:
This won't work correctly, as `switchCase` uses the first parameter as the value to test against (it's trying to use the array in this case).

We should probably consolidate this logic a bit, and roll everything into a single `Utils.cond`, which will remove the duplication in the default case, and also do the right thing in case the covid workspaces are ever moved out of the 'new' column